.github/workflows/test-ui.yml

```diff
@@ -23,6 +23,45 @@ jobs:
         run: cd ./vuu-ui && npm install
       - run: cd ./vuu-ui && npm run test:vite
 
+  cypress-e2e:
+    # As a third party action, cypress-io is pinned to a full length commit SHA for security purposes.
+    # This is also a requirement for the semgrep (static code analysis) scan to pass.
+    # https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions
+    runs-on: ubuntu-latest
```

Comment:
did you consider using windows-latest instead of ubuntu?

Reply:
I went with Ubuntu because that's what all the other actions are using. I guess it could be argued that e2e tests should match an average end-user's environment as close as possible. I don't know if there would be any implication on provisioning different runners though.